OIIInspector/GrpCurlAddressImporter.py

```diff
@@ -0,0 +1,11 @@
+import os
+import logging
+
+LOG = logging.getLogger("OIIInspector")
+
+class GrpCurlAddressImporter:
```

Comment:
I don't see much point in this class. I would argue that the grpcurl binary has to be generally callable in the user's environment (be in PATH)